src/gl/canvas2d.re

```diff
@@ -0,0 +1,76 @@
+type ctx;
+
+/* Canvas API, following https://simon.html5.org/dump/html5-canvas-cheat-sheet.html */
+
+external save : ctx => unit = "save" [@@bs.send];
```

Comment:
You don't have to specify `"save"` since it will be inferred for you based on the name of the `external`. you can just leave it as an empty string: `external save : ctx => unit = "" [@@bs.send];`